### ISSUE TITLE
VAR shall allow to declare several variables (like GLOBAL)

### ISSUE THREAD

**Issue by ldir-hector** [spotlessmind1975/ugbasic#568]
When using OPTION EXPLICIT ON we need to declare each variable on a single line.
It would be more readable if we could type : 
```
VAR tankDroit AS IMAGE, tankGauche AS IMAGE
VAR angle, puissance, dec_x, dec_y, departX, departY
VAR fullscreen AS STRING
```
or maybe 
```
VAR tankDroit, tankGauche AS IMAGE
VAR angle, puissance, dec_x, dec_y, departX, departY
VAR fullscreen AS STRING
```
(but this one could be ambiguous for some users, I know that on some other languages for the first line tankDroit would be a "generic type" and only tankGauche only would be an IMAGE.)

instead of 
```
VAR tankDroit AS IMAGE
VAR tankGauche AS IMAGE
VAR angle
VAR puissance
VAR dec_x
VAR dec_y
VAR departX
VAR departY
VAR fullscreen AS STRING
```

**Comment by spotlessmind1975** [comment]
The improvement has been implemented in the repository, and will be released in the next coldfix. As a matter of syntax unambiguity, each definition has its own story: therefore it is necessary to indicate the type for each variable. However, you can emulate this with the `DEFINE DEAULT TYPE` command:


will be equal to:

    DEFINE DEFAULT TYPE IMAGE
    VAR tankDroit, tankGauche
    DEFINE DEFAULT TYPE WORD
    VAR angle, puissance, dec_x, dec_y, departX, departY
    DEFINE DEFAULT TYPE STRING
    VAR fullscreen

**Comment by spotlessmind1975** [comment]
Released on COLDFIX v1.13 [rev. 20230218.0014]
